style.css

```diff
@@ -0,0 +1,1032 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&family=Quicksand:wght@300;400;500;600;700&display=swap');
+
+* {
```

Comment:
- [ ] I noticed that for this project you use different colors and the design is very different from the original design. Please note that the color for this project are #272A31, #EC5242, #D3D3D3 also make sure that all page elements have similar spacing, borders, shadows, and paddings and exact same colors, for text, backgrounds, borders, etc, from [the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). in order to make the pages should look almost identical to the original design, outside the content differences.